examples/persons/src/FilterByAge.re

```diff
@@ -12,7 +12,7 @@ module PersonsOlderThanQuery = [%graphql
 
 [@react.component]
 let make = (~age) => {
-  let (simple, _full) = useQuery(PersonsOlderThanQuery.make(~age, ()), ());
+  let (simple, _full) = useQuery((module PersonsOlderThanQuery));
```

Comment:
I think you forgot to pass variables too

Reply:
indeed! thanks!